smartsim/settings/dispatch.py

```diff
@@ -378,10 +382,16 @@ class ExecutableProtocol(t.Protocol):
     def as_program_arguments(self) -> t.Sequence[str]: ...
 
 
-class LauncherProtocol(t.Protocol[_T_contra]):
-    def start(self, launchable: _T_contra, /) -> LaunchedJobID: ...
+class LauncherProtocol(collections.abc.Hashable, t.Protocol[_T_contra]):
     @classmethod
+    @abc.abstractmethod
     def create(cls, exp: Experiment, /) -> Self: ...
+    @abc.abstractmethod
+    def start(self, launchable: _T_contra, /) -> LaunchedJobID: ...
+    @abc.abstractmethod
+    def get_status(
+        self, *launched_ids: LaunchedJobID
+    ) -> tuple[SmartSimStatus, ...]: ...
```

Comment:
Not sure how I feel about `start` acting on scalars and `get_status` on collections of `LaunchedJobID`s. I think this may need to be a necessary evil though, especially if we want to enable "batch" status fetching akin to SSv0.7 `SlurmLauncher`.

Probably a good point to discuss with the group!

Reply:
Agreed offline that working over collections is necessary.